collective/easyform/tests/testMailer.py

```diff
@@ -480,9 +480,13 @@ def testNoRecipient(self):
             topic='test subject', replyto='test@test.org', comments='test comments')
 
         self.messageText = ''
-        mailer.onSuccess(request.form, request)
-        self.assertTrue(
-            'mdummy@address.com' in self.mto
+        try:
```

Comment:
assertRaises can be used as context manager (at least with Python 2.7 and/or unittest2) which is probably a bit nicer to read: https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises